compiler-js/src/test/ceylon/dyns/issues_dyns.ceylon

```diff
@@ -330,3 +331,9 @@ void issue5959() {
     check((d of Object) is C5959<String>, "#5959 targs3");
   }
 }
+
+void issue6057() {
+  dynamic DefaultObject {}
+  dynamic { DefaultObject o = eval("this"); }
+  check(identityHash(object extends Basic() {}) > 0, "#6057 1");
```

Comment:
I guess we could leave out the check altogether since we just want to ensure we can call identityHash() without it throwing an error, but I thought why not check it while at it..